tools/lofi/probabilistic_bayesian_update.py

```diff
@@ -209,10 +210,10 @@ def w_dist(
     # Summarize and get shape and scale estimate
     az.summary(trace, hdi_prob=0.90)
 
-    updated_scale = trace.mean()['posterior']['custom_dist']
-    updated_shape = trace.mean()['posterior']['shape']
+    updated_scale = float(trace.mean()['posterior']['custom_dist']) / 10
```

Comment:
Is this supposed to be divided by 100?

Reply:
No this value was interpolated to values in single digit numbers.  This conversion is correct.